layer/executables/packager.py

```diff
@@ -0,0 +1,154 @@
+import inspect
+import os
+import pickle  # nosec
+import shutil  # nosec
+import sys
+import tempfile
+import zipapp
+from pathlib import Path
+from typing import Any, Callable, List, Optional
+from uuid import uuid4
+
+import cloudpickle  # type: ignore
+
+
+def package_function(
```

Comment:
can you please add a test where you call this function multiple times and make sure it works correctly? (there's quite a bit of resource cleanup supposed to happen, just wanna make sure all works well)

Reply:
Can you elaborate what exactly needs to be tested? All the files are stored under the temp dir, which gets deleted once the packaging is done.

Reply:
> All the files are stored under the temp dir, which gets deleted once the packaging is done.

basically wanna see if this holds if the package_function is called multiple times in a row.

Reply:
Got it, I added a test in https://github.com/layerai/sdk/pull/190/commits/2b51c65314afe60fe5ddce0911a210d1f4f97192.